app/scripts/modules/amazon/src/serverGroup/details/sections/LaunchTemplateDetailsSection.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { CollapsibleSection, LabeledValue, LabeledValueList, ShowUserData } from '@spinnaker/core';
+import { IAmazonServerGroupDetailsSectionProps } from './IAmazonServerGroupDetailsSectionProps';
+
+export const LaunchTemplateDetailsSection = ({ serverGroup }: IAmazonServerGroupDetailsSectionProps) => {
+  const { image, launchTemplate, name } = serverGroup;
+  const getBaseImage = (description?: string) => {
+    let name;
+    const tags = (description || '').split(', ');
+    tags.forEach((tag: string) => {
+      const keyVal = tag.split('=');
+      if (keyVal.length === 2 && keyVal[0] === 'ancestor_name') {
+        name = keyVal[1];
+      }
+    });
+
+    return name;
+  };
```

Comment:
Wondering if there's a good place for this utility. I saw something [similar earlier](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/amazon/src/serverGroup/details/sections/LaunchConfigDetailsSection.tsx#L24-L34) and they look very similar.

Reply:
Yeah maybe I will pull it out into a `serverGroup/utils.ts` file.